r/vignettes/dataset.Rmd

```diff
@@ -8,10 +8,10 @@ vignette: >
 ---
 
 Apache Arrow lets you work efficiently with large, multi-file datasets.
-The `arrow` R package provides a `dplyr` interface to Arrow Datasets,
-as well as other tools for interactive exploration of Arrow data.
+The arrow R package provides a [dplyr](https://dplyr.tidyverse.org/) interface to Arrow Datasets,
+and other tools for interactive exploration of Arrow data.
 
-This vignette introduces Datasets and shows how to use `dplyr` to analyze them.
+This vignette introduces Datasets and shows how to use dplyr to analyze them.
 It describes both what is possible to do with Arrow now
 and what is on the immediate development roadmap.
```

Comment:
Does it? I think we should delete this sentence but it'd be good to hear other people's thoughts on this first.

Reply:
At one point this was true (the discussion at the end talked about what's not yet implemented but coming) but perhaps that's no longer accurate.